Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:0.12.4
+
+ADD . /app
+
+WORKDIR /app
+
+RUN npm install
+
+EXPOSE 3000
+VOLUME /app/project-data
+
+ENV SPECS_ALLOW_INSECURE_SSL false
```

Comment:
This is very cool. Do we need another environment variable to set the storage directory or is the local /project-data already mapped to the external volume?